contracts/StakeHub.sol

```diff
@@ -283,45 +281,15 @@ contract StakeHub is SystemV2, Initializable, Protectable {
         uint8,
         bytes calldata msgBytes
     ) external onlyCrossChainContract whenNotPaused enableReceivingFund returns (bytes memory) {
-        (StakeMigrationPackage memory migrationPkg, bool decodeSuccess) = _decodeMigrationSynPackage(msgBytes);
-        if (!decodeSuccess) revert InvalidSynPackage();
-
-        if (migrationPkg.amount == 0) {
-            return new bytes(0);
-        }
-
-        // claim fund from TokenHub
-        bool claimSuccess = ITokenHub(TOKEN_HUB_ADDR).claimMigrationFund(migrationPkg.amount);
-        if (!claimSuccess) {
-            emit MigrateFailed(
-                migrationPkg.operatorAddress,
-                migrationPkg.delegator,
-                migrationPkg.amount,
-                StakeMigrationRespCode.CLAIM_FUND_FAILED
-            );
-            return msgBytes;
-        }
-
-        StakeMigrationRespCode respCode = _doMigration(migrationPkg);
-
-        if (respCode == StakeMigrationRespCode.MIGRATE_SUCCESS) {
-            return new bytes(0);
-        } else {
-            (bool success,) = TOKEN_HUB_ADDR.call{ value: address(this).balance }("");
-            if (!success) revert TransferFailed();
-            emit MigrateFailed(migrationPkg.operatorAddress, migrationPkg.delegator, migrationPkg.amount, respCode);
-            return msgBytes;
-        }
+        revert("deprecated");
```

Comment:
please mark the useless event like `MigrateSuccess` as deprecared

Reply:
fixed

Reply:
marked